en/flight_modes/position_fw.md

```diff
@@ -0,0 +1,27 @@
+# Position Flight Mode (Fixed Wing)

```

Comment:
FYI this is new doc created from the concepts overview and the flight modes summary. It might not be complete, but is no worse than existing content on same topic.